Symfony/src/AppBundle/Resources/views/Register/index.html.twig

```diff
@@ -11,6 +11,11 @@
             <div class="col-md-6 col-md-offset-3">
                 <div class="row">
                     <div class="col-lg-12">
+                        {% if errors %}
```

Comment:
You don't need to check this, `for` in twigs will work despite if the itterable passed has items or not.